libraries/networking/src/udt/SendQueue.cpp

```diff
@@ -305,12 +314,6 @@ void SendQueue::run() {
             sentAPacket = maybeSendNewPacket();
         }
         
-        // Keep track of how long the flow window has been full for
-        if (flowWindowFull && !_flowWindowWasFull) {
```

Comment:
Remove `flowWindowFull` definition above and `_flowWindowWasFull` and `_flowWindowFullSince` in header.